package.json

```diff
@@ -55,7 +62,8 @@
     "jest": "^18.1.0",
     "react": "~15.4.0",
     "react-native": "^0.40.0",
-    "react-native-vector-icons": "^3.0.0"
+    "react-native-vector-icons": "^3.0.0",
+    "shelljs": "^0.7.6"
```

Comment:
Use `yarn` to install `shelljs`

`yarn add shelljs`

It'll also update the `yarn.lock` file.